cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,20 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
```

Comment:
I tried to resort to that, but I experience the circular dependancy issue.

I'll try to think of a way to avoid that issue.